config.rb

```diff
@@ -50,6 +50,7 @@
 
 page '/.htaccess', layout: false
 page '/version.json', layout: false
+page '/meetings.ics', layout: false
```

Comment:
did you forget to commit a new file for this?

Reply:
d'oh